include/boost/math/special_functions/round.hpp

```diff
@@ -14,6 +15,16 @@
 #include <boost/math/policies/error_handling.hpp>
 #include <boost/math/special_functions/math_fwd.hpp>
 #include <boost/math/special_functions/fpclassify.hpp>
+#include <type_traits>
+#include <limits>
+#include <cmath>
+
+#if __cplusplus >= 201703L || _MSVC_LANG >= 201703L
+#include <boost/math/ccmath/ldexp.hpp>
+#  if !defined(BOOST_MATH_NO_CONSTEXPR_DETECTION) && !defined(BOOST_LIBS_MATH_TEST_INSTANTIATE_HPP) // Concept tests throw off constexpr ldexp
```

Comment:
The tests pass for me (msvc and cygwin gcc) without the ` && !defined(BOOST_LIBS_MATH_TEST_INSTANTIATE_HPP)` part, and in general the concept checks are there to catch unexpected bugs that are tricky to spot otherwise.  What was failing without this?

Reply:
This was left over from debugging; I'll remove it.